.circleci/config.yml

```diff
@@ -1,16 +1,23 @@
-version: 2.1
+version: 2
 
 defaults: &defaults
     working_directory: ~/js-threads-client
     docker:
-        - image: circleci/node:10-browsers
+        - image: textile/builder:1.13.1
```

Comment:
just happens to have most of what i needed to test this. node 10 + golang

Reply:
Nice